lighthouse-cli/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
```

Comment:
okay can you followup with @brendankenny then? He's looking at streamlining our release process.

Also, please add developer docs for this to the main readme asap.